x-pack/plugins/kubernetes_security/server/routes/multi_terms_aggregate.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import type { ElasticsearchClient } from '@kbn/core/server';
+import { IRouter } from '@kbn/core/server';
+import { PROCESS_EVENTS_INDEX } from '@kbn/session-view-plugin/common/constants';
+import { MULTI_TERMS_AGGREGATE_ROUTE, AGGREGATE_PAGE_SIZE } from '../../common/constants';
+import {
+  MultiTermsAggregateGroupBy,
+  MultiTermsAggregateBucketPaginationResult,
+} from '../../common/types/multi_terms_aggregate';
+
+export const registerMultiTermsAggregateRoute = (router: IRouter) => {
```

Comment:
Any thoughts about reusing the aggregate route with an extra parameter to differentiate between multi-term group-by and single group-by?

Reply:
I thought about combining them into the same route but when I did that it seemed a little too messy when conditionally building multiple parts of the query body based on the route params. I went with separating those two because of that, even though other than using `multi_term` instead of `term`, and not needing `sort` (in multi term route), they look pretty much the same. I think it will also helps us later because it's much easier to test them when separated.

Reply:
I see, it makes sense now, thanks!